### ISSUE TITLE
I can't seem to get any triplet

### ISSUE THREAD

**Issue by bmd2001** [deltapi/abc-feedback#12]
Hi, I've tried to hit the run button with multiple names but it doesn't seem to find a triplet when I try to do so

**Comment by KaustubhRakhade** [comment]
Firstly, the name has nothing to do with the triples.  You can (and should) use only one name and run as many times as you want. Many of the runs would not yield triples, but eventually you'd find one.
This would no longer be a problem once #3 is resolved. By that time, you can try using the "hacky" code in the comments of #3 to automate the process.

**Comment by JohnyDL** [comment]
Yeah I've had strings of 40+ attempts with no results, but on the other hand I've had a couple of attempts with 40+ results too, it seems that on average a set will have 1 result but that's a very crude average, and the later you started the longer you have to wait for a result... it might be time for some wider attempt pools to increase the chances of a result? (or the automation of restarts #3)

**Comment by deltapi** [comment]
Thanks to all of you for the discussion, I am currently thinking about how to split or pre-scan the workloads in order to get higher chances of finding a triple. Could take some time, though.

**Comment by AaronStephens** [comment]
Consider fetching the next slot if 0 triples are found.   (conditional automation)  The 0s are usually determined very quickly.
